ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -808,6 +808,7 @@ void resetForRetry() {
       logger.trace("BlobId {}: Retry for chunk Id: {}, failed attempts: {}, initializedTime: {}", blobId, chunkBlobId,
           failedAttempts, initializedTimeMs);
       chunkException = null;
+      chunkCompleted = false;
```

Comment:
Is that possible to add the test for the retry?

Reply:
I added in the same test to make sure that there is no retries in this case.